cli/wallet/wallet.go

```diff
@@ -295,6 +389,23 @@ func createWallet(ctx *cli.Context) error {
 	return nil
 }
 
+func signTx(tx *transaction.Transaction, acc *wallet.Account) {
```

Comment:
We do have `SignTx` in RPC client already, there is no need to duplicate it.

Reply:
Or you may make it work with `wallet.Account`, or some other wrapper, I just don't like this code duplication.

Reply:
Done!